packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts

```diff
@@ -18,8 +18,11 @@ export function cleanStandaloneFiles(targetPath: string): Rule {
     tree.delete(posix.join(targetPath, '.husky', 'pre-commit'));
 
     const packageJson = tree.readJson(posix.join(targetPath, 'package.json')) as PackageJson;
-    if (packageJson.scripts?.postinstall) {
-      delete packageJson.scripts.postinstall;
+    if (packageJson.scripts) {
+      packageJson.scripts = Object.fromEntries(Object.entries(packageJson.scripts).filter(([scriptName, _]) => scriptName !== 'postinstall' && scriptName !== 'set:version'));
+    }
+    if (packageJson.devDependencies) {
+      packageJson.devDependencies = Object.fromEntries(Object.entries(packageJson.devDependencies).filter(([depName, _]) => depName !== '@o3r/workspace'));
```

Comment:
The Standalone SDK should not really have dep on `@o3r/workspace`.
Why is it the case?

Reply:
We use a bin from 'o3r/workspace' for the 'set:version'.
https://github.com/AmadeusITGroup/otter/pull/993/files#diff-7c71acfa953343012aa1685726da995972c6c12f718bb3387e60066d54228585

Reply:
Oooh I forgot it, never mind then